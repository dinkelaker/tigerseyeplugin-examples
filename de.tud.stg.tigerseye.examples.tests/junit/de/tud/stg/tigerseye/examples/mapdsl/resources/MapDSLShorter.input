def m2 = [Integer,String: 1="hans"]
