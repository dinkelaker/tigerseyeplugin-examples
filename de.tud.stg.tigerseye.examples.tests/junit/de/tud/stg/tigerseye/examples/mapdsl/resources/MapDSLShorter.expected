def m2 = buildMap(
Integer.class,
String.class,
[
buildEntry(
1,
"hans")
] as Entry[])